README.md

```diff
@@ -1,21 +1,19 @@
 # Homebrew Services
 
-Manage background services with macOS' `launchctl` daemon manager.
+Manage background services using the daemon manager `launchctl` on macOS' or `systemctl` on Linux.
```

Comment:
I don't think the `'` after macOS makes sense anymore, right?

Reply:
Yeah, you're right. That doesn't make sense anymore.